pom.xml

```diff
@@ -468,7 +468,17 @@
     <dependency>
       <groupId>ai.djl.huggingface</groupId>
       <artifactId>tokenizers</artifactId>
-      <version>0.21.0</version>
+      <version>0.18.0</version>
```

Comment:
Could you help to check if we can make this PR work for `0.21.0`?
`0.18.0` has some issues in other env, e.g., CentOS 7, so we bumped up the version to the latest one.

Reply:
Resolved!
Just did some magic to the integration test.